### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by Lui5Jacint0** [ikogan/docker-openmediavault#16]
Hello,

i'm trying to use the docker open media vault on my raspberry pi, but when i run this command:

docker run --name OpenMediaVault -d -ti -h openmediavault.example.test -v /path/to/data:/data:Z -P ikogan/openmediavault

the error shows:

"docker: failed to register layer: Error processing tar file(exit status 4): fatal error: slice bounds out of range

runtime stack:
fatal error: slice bounds out of range
panic during panic

runtime stack:
fatal error: slice bounds out of range
stack trace unavailable"

How may i resolve this?

**Comment by ikogan** [comment]
I don't believe this container will work with an ARM chip. Someone will have to build one based on a Debian arm base image.
